### PR TITLE
Add Review

### PR DESCRIPTION
**Pull request by shikai-zhou**

### CODE REVIEW

**Comment by BenardoTang** [comment]
Can u add the add review feature in UG/DG? not sure how it works. thanks

File: src/test/java/seedu/duke/classes/WatchTimeTest.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
     @Test
     void userReportString() {
         assertEquals("Date : 2020-10-19" + System.lineSeparator()
-                + "Time left today : 60 minutes."
+                + "Time left today : 1 hour(s) 0 minutes."
```
Comment (jazhten) [inline src/test/java/seedu/duke/classes/WatchTimeTest.java:15]: Can I just check if our code can parse hours on top of minutes?

Comment (shikai-zhou) [reply src/test/java/seedu/duke/classes/WatchTimeTest.java:15]: I just made it so that it prints out hours and minutes. Still cannot parse yet.

Comment (jazhten) [reply src/test/java/seedu/duke/classes/WatchTimeTest.java:15]: Ah okay, will raise an issue.

**Pull request merged**